src/main/java/com/alextherapeutics/diga/model/AbstractDigaInsuranceInformation.java

```diff
@@ -33,6 +33,16 @@
      */
     @NonNull
     private String insuranceCompanyIKNumber;
+    /**
+     * The invoice recipients IK number of the insurance that generated this code.
+     */
+    @NonNull
+    private String invoiceRecipientIKNumber;
```

Comment:
this field doesnt seem to be used anywhere? We are using the "IK_des_Rechnungsempfaengers" XML value in invoicing already as the "buyerCompanyCreditorIk" and putting that as the creditor in invoices. I don't think we need it for code validation.

Reply:
True it's not needed at the moment. I removed it